src/components/Menu/Menu/hooks/useLastNavigationDirection.tsx

```diff
@@ -14,21 +14,21 @@ const NAVIGATION_KEYS = [...ARROW_UP_KEYS, ...ARROW_RIGHT_KEYS, ...ARROW_DOWN_KE
 export const useLastNavigationDirection = () => {
   const documentRef = useRef(document);
 
-  const lastNavigationDirectionRef = useRef();
+  const lastNavigationDirectionRef = useRef<NavDirections>();
 
-  const setLastNavigationDirection = useCallback(dir => {
+  const setLastNavigationDirection = useCallback((dir: NavDirections) => {
     lastNavigationDirectionRef.current = dir;
   }, []);
 
   const onKeyEvent = useCallback(
-    ({ key }) => {
-      if (ARROW_UP_KEYS.includes(key)) {
+    (event: KeyboardEvent) => {
```

Comment:
Nitpicky, but you can do:
```suggestion
    ({ key }: KeyboardEvent) => {
```